src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
For now it could be a simple type `Options` which equals to `Object`. Later `Options` can be updated with more specific type. And it will be in one place rather then updating (and possibly missing some) all places in the code that use the same `Options`